js-exercise-4/js/stickyTile/sticky.css

```diff
@@ -0,0 +1,10 @@
+.sticky-product-list {
+  position: relative !important;
+  display: block;
+}
+.sticky-product-group {
```

Comment:
Rank B
Using [CSS element element Selector](https://www.w3schools.com/cssref/sel_element_gt.asp) to increase the priority of your module styles.
Refer to [This article](http://qnimate.com/dive-into-css-specificity/) know what is CSS priority and how to calculate the value.
Ask me if have any unclear point.